[FEATURE] Stage 1 goals and 2 prerequisites (#758) (#762)
TYPO3-Documentation/TYPO3CMS-Guide-CommunityExpansionCommittee: Documentation/Stage-2-Customization/Index.rst

Before you start this stage, please have the following things ready:

> * Have a working local development environment
> * Have a Git repository for your project 
> * Have new TYPO3 project installed using Composer
> * The TYPO3 project is configured and up and running
> * Bootstrap Package extension is installed in the TYPO3 project

Reviewer: ```suggestion
* A working local development environment
* A Git repository for your project 
* A new TYPO3 project
    * Installed using Composer
    * Configured and up and running
    * With the Bootstrap Package extension installed
```
